pkg/openapiclient/local_schemas.go

```diff
@@ -32,7 +32,10 @@ func (k *localSchemasClient) Paths() (map[string]openapi.GroupVersion, error) {
 		return nil, nil
 	}
 	res := map[string]openapi.GroupVersion{}
-	apiGroups, _ := utils.ReadDir(k.fs, filepath.Join(k.dir, "apis"))
+	apiGroups, err := utils.ReadDir(k.fs, filepath.Join(k.dir, "apis"))
```

Comment:
The `apis` folder is allowed to not exist, we should return an error only if the folder exists AND there was an error reading it.